internal/formats/syftjson/to_format_model.go

```diff
@@ -26,6 +30,8 @@ func ToFormatModel(s sbom.SBOM, applicationConfig interface{}) model.Document {
 	return model.Document{
 		Artifacts:             toPackageModels(s.Artifacts.PackageCatalog),
 		ArtifactRelationships: toRelationshipModel(s.Relationships),
+		Files:                 toFile(s),
```

Comment:
More to my point about nailing the distinction. A little weird that `Files` takes the entire sbom, but `Secrets` below it has a subsection from `Artifacts` that we lift to the top level of this struct.

Reply:
> A little weird that Files takes the entire sbom...

you're right, though it's still necessary. The short answer from our offline conversation is that the `Artifacts` struct is not guaranteed to have all of the nodes in the graph that can be created from `sbom.SBOM`, some of the edges in `Relationships` describe nodes that a cataloger may not have directly observed (e.g. a package manager claiming that a package owns a file that does not exist on the filesystem). 

At a minimum to describe all files one needs `sbom.Artifacts` and `sbom.Relationships` to get a full picture of all files that will be in the SBOM. Since this is a helper function of the `sbom` package it seemed right from a consumer point of view to only need to pass an SBOM to get the answer to the question "what files are in my SBOM" (as opposed to needing to pass elements of an SBOM).